typescript/lib/config.ts

```diff
@@ -233,12 +233,40 @@ export class AIConfigRuntime implements AIConfig {
    * Saves this AIConfig to a file.
    * @param filePath The path to the file to save to.
    * @param saveOptions Options that determine how to save the AIConfig to the file.
+   * @param mode Whether to save the AIConfig as JSON or YAML. If unspecified, the file extension will be used to determine the mode.
    */
   public save(
     filePath?: string,
     saveOptions?: SaveOptions,
     mode?: "json" | "yaml"
   ) {
+    const defaultFilePath = mode === "yaml" ? "aiconfig.yaml" : "aiconfig.json";
```

Comment:
Shouldn't this be `<something>.aiconfig.yaml` to include`.aiconfig.` as part of the extension? I guess it's fine for consistency with python side but I had brought it up in https://github.com/lastmile-ai/aiconfig/pull/583 and think it's still worth the discussion